parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -374,6 +381,25 @@ ensureDHCPv6() {
     retrycmd_if_failure 120 5 25 modprobe ip6_tables || exit $ERR_MODPROBE_FAIL
 }
 
+ensureKubeCAFile() {
+    KUBE_CA_FILE="/etc/kubernetes/certs/ca.crt"
+    mkdir -p "$(dirname "${KUBE_CA_FILE}")"
+    echo "${KUBE_CA_CRT}" | base64 -d > "${KUBE_CA_FILE}"
+    chmod 0600 "${KUBE_CA_FILE}"
```

Comment:
```suggestion
    chmod 0644 "${KUBE_CA_FILE}"
```

That's a public key, nothing confidential, might as well let it be accessible.